cms/static/cms/js/build.js

```diff
@@ -171,5 +171,4 @@
          */
         logLevel: 1
     };
-}())  // eslint-disable-line semi
-// A semicolon on the line above will break the requirejs optimizer
+}());
```

Comment:
It now looks like the opposite is happening - lack of a semicolon breaks the build? Which still doesn't make any sense (the semicolon isn't technically required, but is strongly recommended). But at least that's the better way for it to fail.